src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.settings
+
+import com.intellij.openapi.ui.ComboBox
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.FormBuilder
+import org.jdesktop.swingx.JXTitledSeparator
+import javax.swing.JCheckBox
+import javax.swing.JComponent
+import javax.swing.JPanel
+import javax.swing.JTextField
+
+/**
+ * This class displays and captures changes to the values of the Settings entries.
+ */
+class SettingsPluginComponent {
+    var panel: JPanel? = null
+    private var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
+
+    init {
+        panel = FormBuilder.createFormBuilder()
```

Comment:
If you decided to refactor the `init{}` from my other comment, consider refactoring this one in the similar way.

Reply:
Will assess

Reply:
addressed